cSpell.json

```diff
@@ -0,0 +1,70 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": [
+    "Blazor",
+    "BSON",
+    "Cakepop",
+    "chillicream",
+    "Contoso",
+    "customizability",
+    "dataloaders",
+    "discoverability",
+    "entityframework",
+    "evolvability",
+    "fastify",
+    "fetchable",
+    "FIPS",
+    "Hasura",
+    "hotchocolate",
+    "jetbrains",
+    "JWTs",
+    "Linq",
+    "microservices",
+    "middlewares",
+    "NodaTime",
+    "OWIN",
+    "pageable",
+    "protobuf",
+    "roadmap",
+    "runbooks",
+    "struct",
+    "Swashbuckle",
+    "traversion",
+    "Websockets"
+  ],
+  "ignoreWords": [
+    "Specwise",
+    "strawberryshake",
+    "refactorings",
+    "mercurius",
+    "Giroux",
+    "graphyne",
+    "greendonut",
+    "Andi",
+    "aspnetcore",
+    "bananacakepop",
+    "Marek",
+    "Kellner",
+    "Kydne",
+    "GraphiQL",
+    "Tengler",
+    "Staib",
+    "shoooe",
+    "Senn",
+    "Snapshooter",
+    "relayjs",
+    "Rgba",
+    "Postgraphile",
+    "Normen",
+  ],
+  "ignoreRegExpList": [
+    "\\((.*)\\)", // Markdown links
```

Comment:
ℹ️ Technically not correct JSON, but accepted by the tooling and in use in other repositories (I took this from one where I'm currently using it).